download.py

```diff
@@ -51,6 +51,15 @@
             "uv_image/color_variance.txt",
             "uv_image/color.zip",
         ],
+        # NOTE(julieta) encoder data is a framelist and 5 cameras that we hardcode here
+        "encoder": [
+            "frame_list.csv",
```

Comment:
should we consider changing the filename since it might get confused with the frame_list.csv for the decoder?

Reply:
Maybe. I think this is ok because they are under different folders. If we add "encoder" or "decoder" to them then it'd be a bit redudant.